src/modules/__test__/mealscount.test.js

```diff
@@ -0,0 +1,17 @@
+const totalMeals = require('../countMeals.js');
+
+describe('Test the count of Meals on the Homepage', () => {
+  test('Should count all meals displayed on the page and return the count', () => {
+    const mealsarray1 = [
+      { strMeal: 'Banana Pancakes', strMealThumb: 'https://www.themealdb.com/images/media/meals/sywswr1511383814.jpg', idMeal: '52855' },
+      { strMeal: 'BBQ Pork Sloppy Joes', strMealThumb: 'https://www.themealdb.com/images/media/meals/atd5sh1583188467.jpg', idMeal: '52995' }];
+
+    const Count = totalMeals(mealsarray1);
```

Comment:
[OPTIONAL] As per project requirement, you need to test is with a visible generated element on the dom. You can count them with their HTML tag, their class etc with querySelector. But first you need two fix your counter functions.